rotkehlchen/chain/evm/decoding/thegraph/decoder.py

```diff
@@ -58,6 +62,52 @@ def _decode_delegator_staking(self, context: DecoderContext) -> DecodingOutput:
             return self._decode_stake_locked(context)
         elif context.tx_log.topics[0] == TOPIC_STAKE_DELEGATED_WITHDRAWN:
             return self._decode_stake_withdrawn(context)
+        elif context.tx_log.topics[0] == DELEGATION_TRANSFERRED_TO_L2:
+            return self._decode_delegation_transferred_to_l2(context)
+        return DEFAULT_DECODING_OUTPUT
+
+    def _decode_delegation_transferred_to_l2(self, context: DecoderContext) -> DecodingOutput:
+        """Decode a delegation transfer to L2 event"""
```

Comment:
This kind of comment is not useful since it is repeating the docstring. An exampel would be

```suggestion
    def _decode_delegation_transferred_to_l2(self, context: DecoderContext) -> DecodingOutput:
        """Decode a transfer delegating GRT from ethereum to arbitrum"""
```

Reply:
Got it